### ISSUE TITLE
Label/Legend in chart component

### ISSUE THREAD

**Issue by uwekaditz** [knowthelist/ftui#38]
Using the label with the following code `<ftui-chart-data label="(Min: $min °C / Max: $max °C)" color="gray">` works fine as label
![grafik](https://user-images.githubusercontent.com/55511593/179825840-e9711a7c-11ce-4370-96ba-e88d532501dd.png)
but the same code is also used while hovering the chart data but the **_$functions_** are not resolved.
![grafik](https://user-images.githubusercontent.com/55511593/179826350-8b0cda8e-bd9a-4dbf-8c44-8a99731f8bab.png)

**Comment by knowthelist** [comment]
yes, not cool. But to change this a tooltips callback is necessary 

options: {
        tooltips: {
          callbacks: {
             label: function(tooltipItem, data) {
                return ...;
             }
          }
       }
